.eslintrc

```diff
@@ -38,7 +38,7 @@ rules:
   class-methods-use-this: off
   react/jsx-tag-spacing: ["error", { "closingSlash": "never", "beforeSelfClosing": "allow", "afterOpening": "never" }]
   react/forbid-prop-types: off
-  react/jsx-filename-extension: [1, { "extensions": [".js", ".jsx"] }]
+  react/jsx-filename-extension: [1, { "extensions": [".js", ".jsx", ".ts", ".tsx"] }]
```

Comment:
This addition should probably be part of the previous commit (bbbc054990481613b441f2690d0657d5494cfea2) and not lumped in with the example TypeScript so it's retained instead of dropped before merge.

Reply:
Thanks - changed!

For what it's worth, I think the first commit is still incomplete and will need some webpack config / ts config etc changes to get it working smoothly.